src/drf_pydantic/parse.py

```diff
@@ -135,6 +135,10 @@ def _convert_field(field: pydantic.fields.FieldInfo) -> serializers.Field:
     if _default_value is not pydantic_core.PydanticUndefined:
         drf_field_kwargs["default"] = _default_value
 
+    # Adding description as help_text
+    if field.description:
```

Comment:
I would instead check for two things explicitly: `None` or `pydantic_core.PydanticUndefined`. The truthy check is a bit ambiguous.

Reply:
Resolved. Added both, so serializer field didnt look like this without description:
CharField(allow_null=True, default=None, help_text='None, required=False)

Reply:
I added test for fields into 'test_drf_field_kwargs'